extensions/native/circuit/src/fri/mod.rs

```diff
@@ -177,7 +179,29 @@ impl<F: Field> BaseAir<F> for FriReducedOpeningAir {
     }
 }
 
-impl<F: Field> BaseAirWithPublicValues<F> for FriReducedOpeningAir {}
+impl FriReducedOpeningAir {
+    pub fn columns<F: Field>(&self) -> Vec<String> {
+        PrefixCols::<F>::flatten_fields()
```

Comment:
Not sure about this one

Reply:
But if it is correct, should we prefix some specific value to each column to be able to differentiate them? Like instr1_, instr2, workload_, etc?